Makefile

```diff
@@ -63,6 +54,15 @@ all: build
 
 include versioning.mk
 
+BUILD_TAG :=
```

Comment:
maybe add a comment here:

```bash
# This should be located after the include directive for versioning.mk,
# so that we can override DOCKER_IMG_VERSION with CUSTOM_BUILD_TAG.
```